src/main/java/net/pod/peaengine/render/texture/SpriteRegistry.java

```diff
@@ -0,0 +1,12 @@
+package net.pod.peaengine.render.texture;
```

Comment:
Please add a private default constructor and a way go get registry entries

Reply:
It would also be good to make a base class Registry<> to avoid repeated code in future